types/svgo/index.d.ts

```diff
@@ -314,6 +314,7 @@ export type PluginsPlugin = DefaultPlugin<'plugins'>;
 export type PrefixIdsPlugin = DefaultPlugin<
     'prefixIds',
     {
+        prefix?: boolean | string | ((node: any, info: any) => string),
```

Comment:
```suggestion
        prefix?: boolean | string | ((node: any, info: any) => string) | undefined;
```
I believe this should work at runtime, to keep with convention here and to get TS4.4 strict optional properties support when used

Reply:
Added your suggestion, thanks!